README.md

```diff
@@ -40,21 +40,28 @@ Check out the [getting started guide](./docs/getting-started.md) for launching a
-|DigitalOcean Provider v1alpha1 (v0.2)|✓|✓|✓|||||
-|DigitalOcean Provider v1alpha2 (v0.3)||✓|✓|✓|✓|||
-|DigitalOcean Provider v1alpha3 (v0.4)||||✓|✓|✓|✓|
+|                 | DigitalOcean Provider v1alpha1 (v0.1) | DigitalOcean Provider v1alpha1 (v0.2) | DigitalOcean Provider v1alpha2 (v0.3) | DigitalOcean Provider v1alpha3 (v0.4) | DigitalOcean Provider v1alpha4 (v0.5) | DigitalOcean Provider v1 (v1.0) |
+| --------------- | ------------------------------------- | ------------------------------------- | ------------------------------------- | ------------------------------------- | ------------------------------------- | ------------------------------- |
+| Kubernetes 1.13 | ✓                                     | ✓                                     |                                       |                                       |                                       |                                 |
+| Kubernetes 1.14 | ✓                                     | ✓                                     |                                       |                                       |                                       |                                 |
+| Kubernetes 1.15 | ✓                                     | ✓                                     |                                       |                                       |                                       |                                 |
+| Kubernetes 1.16 |                                       |                                       | ✓                                     |                                       |                                       |                                 |
+| Kubernetes 1.17 |                                       |                                       | ✓                                     | ✓                                     |                                       |                                 |
+| Kubernetes 1.18 |                                       |                                       |                                       | ✓                                     |                                       | ✓(?)                            |
+| Kubernetes 1.19 |                                       |                                       |                                       | ✓                                     |                                       | ✓(?)                            |
+| Kubernetes 1.20 |                                       |                                       |                                       | ?                                     |                                       | ✓(?)                            |
+| Kubernetes 1.21 |                                       |                                       |                                       | ?                                     | ?                                     | ✓(?)                            |
 
```

Comment:
you can add 1.22 and 1.23 as well both works in the 1.0
also v1 should work in 1.19/1.20 as well

not sure in 1.18 but that is already deprecated so we can remove that out